packages/commonjs/test/snapshots/function.js.md

```diff
@@ -179,6 +142,12 @@ Generated by [AVA](https://ava.li).
       var foo_bar = 'fubar';␊
       ␊
       t.is(foo_bar, 'fubar');␊
+      ␊
+      var main = {␊
+      ␊
+      };␊
+      ␊
+      module.exports = main;␊
```

Comment:
Practically there's no difference here right? As it's an empty object vs the default empty exports object...

Reply:
Good point. This could be an optimization to add.

Reply:
Actually, this could be problematic to get rid of because this plugin converts to ESM and that one does not have an implicit default export. So an empty object is generated. I do not see how Rollup core could get rid of this easily later. I think this would be a job to be fixed manually via an `exposedExports` option.